src/test/java/teammates/test/cases/BaseComponentTestCase.java

```diff
@@ -106,9 +106,21 @@ protected static void verifyPresentInDatastore(FeedbackQuestionAttributes expect
         if (wildcardId) {
             expected.setId(actual.getId());
         }
+        
+        setEqualTimestampBetweenExpectedAndActual(expected, actual);
+        
```

Comment:
We need the production method for testing later. I am following the approach as in #4657 

When we create a new FeedbackQuestion for testing later, we need to actually return a default timestamp when the createdAt and updatedAt are null. But once the timestamp field is added, we do not have a way to set them to null, hence we might need the methods for testing purposes, as in #4695 

This is the way that I can think of. If you have a better idea, I am open to that as well.